pkg/subctl/cmd/gather/cni.go

```diff
@@ -0,0 +1,161 @@
+/*
+© 2021 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gather
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/resource"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	typeIPTables = "iptables"
+	typeOvn      = "ovn"
+	typeUnknown  = "unknown"
+	libreswan    = "libreswan"
+)
+
+var ipCmds = map[string]string{
+	"ip-routes":         "ip route show",
+	"ip-rules":          "ip rule list",
+	"ip-rules-table150": "ip rule show table 150",
```

Comment:
```suggestion
	"ip-rules-table150": "ip route show table 150",
```

Reply:
Oh sorry, I asked him to change that, we need both.  OVN uses rules in table 150.

Reply:
@vthapar can we add both?